boskos/aws-janitor/cmd/aws-janitor-boskos/main.go

```diff
@@ -34,6 +34,8 @@ import (
 
 var (
 	boskosURL          = flag.String("boskos-url", "http://boskos", "Boskos URL")
+	username           = flag.String("username", "", "Username used to access the Boskos server")
```

Comment:
We want to validate that it you set one flag, you set both

Reply:
Done.